Xamarin.Forms.Core/FillRule.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Xamarin.Forms
+{
+    public enum FillRule
```

Comment:
What's this?

Reply:
A value that determines how the intersecting areas are combined. 
https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.pathgeometry.fillrule?view=netcore-3.1